client/v2/internal/testpbpulsar/msg.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
 
-package testpb;
+package testpbpulsar;
```

Comment:
Renamed, because test was failing (on ci only) for double registration (https://github.com/cosmos/cosmos-sdk/actions/runs/15034823230/job/42254518136 / https://github.com/cosmos/cosmos-sdk/blob/f78f0f4fbdd732ae525d30216f6924d32ac621f5/testutil/testdata/testpb/tx.proto#L13)